### PR TITLE
Move 'add category to favourites' JS code into catetogy module

### PR DESCRIPTION
**Pull request by nop33**

### CODE REVIEW

**Comment by nop33** [comment]
Or just cherry-pick the commit and cancel the PR

File: indico/htdocs/js/indico/modules/categories/display.js [hunk 1]
```diff
@@ -68,3 +68,42 @@ function setupCategoryDisplayEventList(show_past_events) {
...
+            delay: 500
+        },
+        content: {
+            text: function() {
+                var $this = $(this);
+                if ($this.hasClass('enabled')) {
+                    return $T.gettext("Remove from your favourites");
+                } else {
+                    return $T.gettext(
+                        '<h3>Add to your favourites</h3>' +
```
Comment (ThiefMaster) [inline indico/htdocs/js/indico/modules/categories/display.js:102]: should be two separate gettext calls

Comment (nop33) [inline indico/htdocs/js/indico/modules/categories/display.js:102]: Is the commit merged already?

Comment (ThiefMaster) [inline indico/htdocs/js/indico/modules/categories/display.js:102]: at least not in devel

**Pull request closed without merging**